addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1663,7 +1663,7 @@ var AbstractFieldBinary = AbstractField.extend({
         this._super.apply(this, arguments);
         this.fields = record.fields;
         this.useFileAPI = !!window.FileReader;
-        this.max_upload_size = 25 * 1024 * 1024; // 25Mo
+        this.max_upload_size = odoo.session_info.max_file_upload_size || 128 * 1024 * 1024;
```

Comment:
```suggestion
        this.max_upload_size = session.max_file_upload_size || 128 * 1024 * 1024;
```
That way, you should no longer need to patch anything in the tests